app/templates/docs/erediensten-toezichtsdatabank-sparql.hbs

```diff
@@ -0,0 +1,366 @@
+<AuHeading @skin="1" @level="1">Erediensten Toezichtsdatabank SPARQL API <em>[CONCEPT]</em></AuHeading>
+
+<p>
+  Via deze API is het mogelijk om SPARQL query's uit te voeren op de Toezichtsdatabank voor Erediensten waar besluiten te vinden zijn die naar het Loket zijn gepubliceerd. Deze API is gebaseerd op de <LinkTo @route="docs.vendor-sparql">Vendor SPARQL API</LinkTo>.
+</p>
+
+<AuHeading @level="2" @skin="2">Verbinden</AuHeading>
+<p>
+  De endpoint voor deze API kan bereikt worden via
+</p>
+<CodeBlock @code="https://databankerediensten.lokaalbestuur.vlaanderen.be" />
+<p>
+  Ter volledigheid staat hieronder de volledige lijst aan URL's voor het inloggen, sturen van query's en het uitloggen respectievelijk.
+</p>
+<CodeBlock @code="https://databankerediensten.lokaalbestuur.vlaanderen.be/vendor/login
+https://databankerediensten.lokaalbestuur.vlaanderen.be/vendor/sparql
+https://databankerediensten.lokaalbestuur.vlaanderen.be/vendor/logout" />
+<p>
+  <strong>Bekijk zeker de pagina over de <LinkTo @route="docs.vendor-sparql">Vendor SPARQL API</LinkTo> om te leren hoe de SPARQL API werkt!</strong>
+</p>
+
+<AuHeading @level="2" @skin="2">Sturen van query's</AuHeading>
+<p>
+  Op dit endpoint kunnen alle gegevens omtrent inzendingen teruggevonden worden, samen met het ingevulde formulier voor de publicatie in Loket. Deze formuliergegevens zitten in bestanden die via een downloadlink kunnen worden gedownload. Al deze informatie is vrij te ondervragen in deze SPARQL endpoint met de volledige vrijheid van SPARQL voor filteren op datum, URI, regex, &hellip;
+</p>
+
+<AuHeading @level="3" @skin="3">Voorbeelden</AuHeading>
+<p>
+  Dit eerste eenvoudig voorbeeld geeft een korte lijst van een aantal willekeurige inzendingen (Submissions); een query voor het ontdekken van een aantal URI's die in volgende query's gebruikt kunnen worden.
```

Comment:
Should this be `eenvoudige` instead of `eenvoudig`?

Reply:
Though not incorrect, the form with -e seems to be preferred. Will change it. Omitting the -e can be seen as a stylistic choice. [Dutch can be difficult.](https://www.vlaanderen.be/team-taaladvies/taaladviezen/bijvoeglijke-naamwoorden-verbuigen-verbuiging-met-of-zonder-e)